### ISSUE TITLE
Support Open Remote Repo in GitHub Code menu

### ISSUE THREAD

**Issue by devhawk** [microsoft/vscode-remote-repositories-github#24]
Can "Open as Remote Repository in VSCode" be added to this menu? I suspect this might need to be done w/ an extension. 

![image](https://user-images.githubusercontent.com/8965/122446015-0c46c100-cf57-11eb-9157-4c7627ffb6d0.png)

When I'm logged in, there's also an "Open with Visual Studio" option. 

![image](https://user-images.githubusercontent.com/8965/122449521-c8ee5180-cf5a-11eb-8d5a-e723117f43ad.png)

**Comment by JustinGrote** [comment]
I mean if Open with VIsual Studio is allowed then I don't see why "Visual Studio Code" wouldn't be.

It should have a URL handler, e.g. vscode-git://path/to/my/repo that would work with other providers e.g. gitlab/etc.

**Comment by jianboy** [comment]
good idea.

**Comment by JustinGrote** [comment]
@jianboy it does have a remote URL handler as discussed in https://github.com/microsoft/vscode-remote-repositories-github/issues/2#issuecomment-865399969, so as a workaround you can put this in your repo README for others to use.

**Comment by jianboy** [comment]
In fact, this question is not about the function of ”vscode-remote-repositories-github“ plugin, but he suggest github website add a drop-down box. so you can use vscode to open any project.

@JustinGrote 

Adding a VSCode link in README.md is a good idea. But you should know all the current github repositories don't have the URL. So adding a drop-down box is the best way.

![image](https://user-images.githubusercontent.com/11532015/123003942-814a4880-d3e6-11eb-8443-0a2ccb68a32f.png)

**Comment by JustinGrote** [comment]
@jianboy agreed having it in the github menu is ideal, my recommendation is just a stop-gap workaround.

**Comment by jianboy** [comment]
**We don't need to install vscode at all.**
githuub can add a vscode remote button.

![image](https://user-images.githubusercontent.com/11532015/123004676-83f96d80-d3e7-11eb-8d73-0913859c7426.png)

and browsing the code online is the best solution. And there are mature similar solutions.

**Comment by JustinGrote** [comment]
So you're basically asking for github1s.com to be natively integrated into Github it sounds like :)

**Comment by jianboy** [comment]
@JustinGrote 

Great minds think alike.

**Comment by JustinGrote** [comment]
@jianboy I made a tampermonkey script to do this.
![image](https://user-images.githubusercontent.com/15258962/123886232-dd8bfb80-d903-11eb-8455-6e280c70f881.png)

https://gist.github.com/JustinGrote/0148f9accc7355f650cd0a0f6b404fa5

**Comment by LinqLover** [comment]
Not the official VS Code extension, but very nice idea! ❤️ Maybe you could even move the icon into the "Code" menu as proposed by @jianboy?

**Comment by Sidarta85** [comment]
> @jianboyEu fiz um script tampermonkey para fazer isso. ![imagem](https://user-images.githubusercontent.com/15258962/123886232-dd8bfb80-d903-11eb-8455-6e280c70f881.png)
> 
> https://gist.github.com/JustinGrote/0148f9accc7355f650cd0a0f6b404fa5

**Comment by aetonsi** [comment]
> @jianboy I made a tampermonkey script to do this. ![image](https://user-images.githubusercontent.com/15258962/123886232-dd8bfb80-d903-11eb-8455-6e280c70f881.png)
> 
> https://gist.github.com/JustinGrote/0148f9accc7355f650cd0a0f6b404fa5

Doesn't seem to work anymore :( i tried to fix it but i couldn't..

**Comment by JustinGrote** [comment]
@aetonsi I'll take a look if I get a moment, it would be nice to have a first-party button for this though.

**Comment by aetonsi** [comment]
@joyceerhl sorry for bothering you, also because i know that altering the github.com website is not the scope of this repository, and that probably it is up to another Microsoft team to implement such modifications, but, as far as you know, is there any possibility at all that this will happen?

I think the little kid vscode is all grown up now, and ready to be alongside its father Visual Studio (_i know they're not related_) in the "⤓ Code" sub menu... (jk)

**Comment by aetonsi** [comment]
In the meantime i updated @JustinGrote 's script: 
> https://gist.github.com/aetonsi/26d4257e439a94dfacb5b230fb4b438a

I added a "clone" button.
The "open" button doesn't, since i don't know which `vscode://` URL to open in order to emulated VSCode's `Open Remote Repository`/`Open Repository from Github` functionality. I tried with `vscode-vfs://` URLs, no luck..
Discussion here:
> https://gist.github.com/JustinGrote/0148f9accc7355f650cd0a0f6b404fa5?permalink_comment_id=4597081#gistcomment-4597081
